src/index.ts

```diff
@@ -1,71 +1,29 @@
 import { spawn } from 'child_process'
-import { join } from 'path'
 import { createRequire } from 'module'
 import { pathToFileURL } from 'url'
-import { readdir } from 'fs/promises'
+import { resolveTestPaths } from './fs.js'
 
-const SUPPORTED_EXTENSIONS = ['.js', '.mjs', '.cjs', '.ts', '.mts', '.cts']
+const DEFAULT_SUPPORTED_EXTENSIONS = ['.js', '.mjs', '.cjs', '.ts', '.mts', '.cts']
+
+export function getSupportedExtensions (): string[] {
+  if (typeof process.env.SUPPORTED_EXTENSIONS !== 'undefined' && process.env.SUPPORTED_EXTENSIONS !== null) {
+    return process.env.SUPPORTED_EXTENSIONS.split(',').map(e => e.trim())
```

Comment:
Yep, sounds good. I'll rename the environment variable to `TEST_EXTENSIONS` because that makes more sense, and I'll also rename the function to `getTestExtensions()` for the same reason.